OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -175,8 +175,8 @@ public interface INotifyDeployComplete
 
 	public interface INotifyDeployTriggered
 	{
-		void Deploy(Actor self, bool skipMakeAnim);
-		void Undeploy(Actor self, bool skipMakeAnim);
+		void Deploy(Actor self, bool skipMakeAnim, string[] bodyNames);
+		void Undeploy(Actor self, bool skipMakeAnim, string[] bodyNames);
```

Comment:
If I were to have DeployTypes instead of bodyNames, we could remove bool skipMakeAnim and let have DeployTypes of length = 0 to skip make anims. Do you prefer to keep skipMakeAnim?